test/spec/browser_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'yaml'
+require_relative './app_spec.rb'
+
+config = YAML.load_file('config.yml')
+webdriver_url = "http://#{config['sauce_username']}:#{config['sauce_access_key']}@ondemand.saucelabs.com:80/wd/hub"
+
+browsers = [
+  :chrome,
+  {
+    appiumVersion: '1.6.3',
+    deviceName: 'iPad Simulator',
+    platformName: 'iOS',
+    platformVersion: '10.0',
+    browserName: 'Safari',
+    javascriptEnabled: true
+  }
```

Comment:
Starting with an iPad because it's easier to debug due to the larger screen. Once it passes let's add iPhone.